language/Methods.md

```diff
@@ -105,16 +105,23 @@ DoSomething(7, 8, 9, 10);
 *actions*, which are intended to be run from actor states and give extra
 information to the function. Action functions change the meaning of the `self`
-parameter and pass in `invoker` and `stateinfo` parameters as well. `stateinfo`
-refers to the `State` which this action was called from. Here is a chart for
-the meanings of the `self` and `invoker` variables under each scope:
-
-| Scope     | `self` meaning                                                  | `invoker` meaning |
-| -----     | --------------                                                  | ----------------- |
-| None      | The actor this function operates on, ambiguous in some contexts | State owner       |
-| `actor`   | The actor                                                       | The actor         |
-| `item`    | Item owner                                                      | Item itself       |
-| `overlay` | Weapon owner                                                    | Weapon itself     |
-| `weapon`  | Weapon owner                                                    | Weapon itself     |
+parameter and may pass in `invoker` and `stateinfo` parameters as well. `stateinfo`
+refers to the `State` which this action was called from.
+
+Action scopes are a flag field and as such an action function can have multiple scopes.
+The only scopes that actually set the internal action function flag are `item`, `overlay` and `weapon`,
+so action functions without one of these do not act much like action functions, and they do not get an
+`invoker` or `stateinfo` pointer.
+
+Here is a chart for the meanings of the `self` and `invoker` variables under each scope.
+Scopes lower down on the chart "override" scopes above them, so to speak:
+
+| Scope     | `self` meaning                                                                                                               | `invoker` meaning |
+| -----     | --------------                                                                                                               | ----------------- |
+| None      | The actor this function operates on, ambiguous in some contexts                                                              | N/A               |
+| `actor`   | The actor                                                                                                                    | N/A               |
+| `item`    | Context-dependent. The item itself when it exists in-world, but the owner when used as part of a CustomInventory state chain | Item itself       |
+| `overlay` | Context-dependent. The actor itself when it exists in-world, but the owner of the PSprite when it is being used as one       | Context-dependent. The actor itself when it exists in-world, but the PSprite's `caller` field when it is being used as one. In most cases this field should be set to be the actor defining the action function, but it can be changed by user code so this is not technically guaranteed |
```

Comment:
PSprite is a technical term, which while correct, is not very intuitive. Preferably "the weapon" would be used here because it is an equivalent term in abstract.

Reply:
> 
> 
> PSprite is a technical term, which while correct, is not very intuitive. Preferably "the weapon" would be used here because it is an equivalent term in abstract.

That's not actually correct. PSprites don't have to be weapons, and don't have to be contained in them.